h/auth/role.py

```diff
@@ -1,4 +1,5 @@
 #: Administrators. These users have super cow powers.
+# pylint: disable=invalid-name
```

Comment:
:+1: These are class name type things. These are probably going to get removed / replaced with an `Enum` soon.